src/pages/users/register/RegisterStepTwoStoreInfo.jsx

```
@@ -0,0 +1,49 @@
+          />
+        ) : (
+          <TextInput
+            key={idx}
+            type={field.type}
+            value={values[idx]}
+            placeholder={field.placeholder}
+            onChange={(e) => handleChange(idx, e.target.value)}
+          />
+        )
+      )}
+      <Button children="다음" onClick={() => onNext(values)} disabled={!allFilled} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)